includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Advanced Settings Fields file.
+ *
+ * @package ActivityPub
+ */
+
+namespace ActivityPub\WP_Admin;
+
+/**
+ * Advanced Settings Fields class.
+ */
+class Advanced_Settings_Fields {
+
+	/**
+	 * Initialize.
+	 */
+	public static function init() {
+		add_action( 'load-settings_page_activitypub', array( self::class, 'register_advanced_fields' ) );
+	}
+
+	/**
+	 * Register settings.
+	 */
+	public static function register_advanced_fields() {
+		\add_settings_section(
+			'activitypub_advanced_settings',
+			\__( 'Advanced Settings', 'activitypub' ),
+			array( self::class, 'render_advanced_settings_section' ),
+			'activitypub_advanced_settings'
+		);
+
+		add_settings_field(
+			'activitypub_outbox_purge_days',
+			__( 'Outbox Retention Period', 'activitypub' ),
+			array( self::class, 'render_outbox_purge_days_field' ),
+			'activitypub_advanced_settings',
+			'activitypub_advanced_settings',
+			array( 'label_for' => 'activitypub_outbox_purge_days' )
+		);
+
+		\add_settings_field(
+			'activitypub_authorized_fetch',
+			__( 'Authorized Fetch', 'activitypub' ),
+			array( self::class, 'render_authorized_fetch_field' ),
+			'activitypub_advanced_settings',
+			'activitypub_advanced_settings',
+			array( 'label_for' => 'activitypub_authorized_fetch' )
+		);
+	}
+
+	/**
+	 * Render Advanced Settings Section.
+	 */
+	public static function render_advanced_settings_section() {
+		?>
+		<p>
+			<?php
+			$allowed_html = array(
+				'a' => array(
+					'href'   => true,
+					'target' => true,
+				),
+			);
+			echo wp_kses( __( 'Advanced settings allow deep customization but can affect your site&rsquo;s functionality, security, or performance if misconfigured. Only proceed if you fully understand the changes, and always back up your site beforehand. If unsure, consult <a href="https://github.com/Automattic/wordpress-activitypub/tree/trunk/docs" target="_blank">documentation</a> or seek <a href="https://wordpress.org/support/plugin/activitypub/" target="_blank">expert advice</a>.', 'activitypub' ), $allowed_html );
```

Comment:
```suggestion
			echo wp_kses( __( 'Advanced settings allow deep customization but can affect your site&#8217;s functionality, security, or performance if misconfigured. Only proceed if you fully understand the changes, and always back up your site beforehand. If unsure, consult <a href="https://github.com/Automattic/wordpress-activitypub/tree/trunk/docs" target="_blank">documentation</a> or seek <a href="https://wordpress.org/support/plugin/activitypub/" target="_blank">expert advice</a>.', 'activitypub' ), $allowed_html );
```
Just to double-check, we're fine with making the URLs translatable?
I don't know if it's a convention or something others have done for security purposes, but I've mostly seen URLs formatted in to avoid translators accidentally breaking them, unless there was a localized version, like for `https://wordpress.org/support/plugin/activitypub/` for example.
Not sure what's the most right thing here, just want to make sure we make that decision consciously.